tdex-daemon.md

```diff
@@ -116,6 +116,21 @@ $ tdex --help
 
 Now you are ready to [deposit funds](#deposit-funds) to create your first market and start accepting incoming trades.
 
+## NEW: Run with Elements node as explorer
+
+It is now possible to connect the daemon directly to an Elements node instead of sourcing blockchain data from the default electrs explorer.
+You can use the `TDEX_ELEMENTS_RPC_ENDPOINT` env var to set the endpoint for connecting the daemon to the node. Currently, only insecure connection (no TLS encryption) is available.
+In the special case you need to restore an already used wallet, you can also make use of the `TDEX_ELEMENTS_START_RESCAN_TIMESTAMP` to set the node's starting point of the rescan.
+If you'll create a brand new wallet you can leave that variable unset. Be aware that the restore phase can take up to several hours to complete, depending how far in the past you set the rescan timestamp. You can even rescan the entire blockchain using the `0` value, but this is highly discouraged, 
+
+### Run
+
+```sh
+export TDEX_ELEMENTS_RPC_ENDPOINT="http://admin:pass@127.0.0.1:8332"
```

Comment:
I would be consistent and move this examples in the big chunk in the "Run" section at line 8.

Also do the same for Docker section.

The long note about the new explorer type should stay at the beginning IMHO at line 8 and mentioning the v0.2.0 is the first versions supporting this.